<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuOption</class>
 <widget class="QDialog" name="MenuOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> User Option Menu</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>261</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="1" column="0">
     <widget class="QGroupBox" name="support">
      <property name="title">
       <string>2) Support</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="supportButton">
         <property name="text">
          <string>Support</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="profil">
      <property name="title">
       <string>3) View profil</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="showProfilButton">
         <property name="text">
          <string>Show</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+Shift+P</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="ranking">
      <property name="title">
       <string>4) Consult ranking</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>37</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="showRankingButton">
         <property name="text">
          <string>Show</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+R</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QGroupBox" name="friendMenu">
      <property name="title">
       <string>5) Consult friend</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="2">
        <widget class="QPushButton" name="showFriendButton">
         <property name="text">
          <string>Show</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+F</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>41</width>
           <height>37</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QGroupBox" name="deconnexion">
      <property name="title">
       <string>6) Deconnexion</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="2">
        <widget class="QPushButton" name="leaveButton">
         <property name="text">
          <string>Leave Account</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+D</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>41</width>
           <height>37</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="play">
      <property name="title">
       <string>1) Play a party</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="4" column="0">
        <widget class="QPushButton" name="playButton">
         <property name="text">
          <string>Play</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+P</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="timeButton">
         <property name="text">
          <string>Time Trial Mode</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="vsButton">
         <property name="text">
          <string>Vs Mode</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="teamButton">
         <property name="text">
          <string>Team Mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>960</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>main.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="gifLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>301</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>test.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>301</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>title2.gif</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>gifLabel</zorder>
  <zorder>titleLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
